parsers/BD.py

```diff
@@ -75,9 +73,10 @@ def new_format_converter(df, logger):
     return df
 
 
-def production_processer(df, target_datetime, old_format=False):
-    """Takes dataframe and extracts all production data and timestamps.
-    Returns a list of 2 element tuples in form (dict, arrow object)."""
```

Comment:
The second line of this comment is not fully captured in the return type annotation. Same below for exchange_processer

Reply:
I think that it can be done using __future__
Needs testing